README.md

```diff
@@ -41,6 +36,81 @@ python -m dap_rt_reporter\
   --desc tests/integration/resources/simple_test_config.csv --log execute.log
 ```
 
+## Usage
+
```

Comment:
Clarify that when used as library report specification should be done programatically.
And when used as a program the report specification shall be given as a CSV